### PR TITLE
Update fileparser.py

### PR DESCRIPTION
**Pull request by barisccolak**
Update code and action to solve error in the workflow.

### CODE REVIEW

File: testmodule/jobfile.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        self.file_path = file_path
+        self.file_name = os.path.basename(file_path)
+        self.foldername = None
+        self.lines = None
+        self.headlines = []
+        self.programlines = []
+        self.separator = None
+        self.error_flag = False
+
+        self.warnings = []
```
Comment (CagtayFabry) [inline testmodule/jobfile.py:21]: I don't think you are using this anymore (`self.warnings`)

File: testmodule/main.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    """Receive single input file."""
+    job = JobFile(file_path)
+
+    for rule in rules:
+        result = rule.apply_rule(job)
+        if result is not None:
+            warning = (
+                result[0] + str(result[1]) + " [" + str(result[2]) + "] : " + result[3]
+            )
+            print(warning)
```
Comment (CagtayFabry) [inline testmodule/main.py:73]: Check if `input_folder` and `input_file` are still relevant for the code, otherwise remove

File: testmodule/EXfileparser.py
Comment (CagtayFabry) [inline testmodule/EXfileparser.py:1]: remove old file

File: testmodule/jobfile.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    def __init__(self, file_path: str):
+        """Initialize."""
+        self.file_path = file_path
+        self.file_name = os.path.basename(file_path)
+        self.foldername = None
+        self.lines = None
+        self.headlines = []
+        self.programlines = []
+        self.separator = None
+        self.error_flag = False
```
Comment (CagtayFabry) [inline testmodule/jobfile.py:19]: I don't think you are using this anymore (`self.error_flag`)

File: testmodule/main.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+def main():
+    file_path = "/mnt/scratch/bcolak/Internship/testmodule/test.JBI"
```
Comment (CagtayFabry) [inline testmodule/main.py:76]: use relative path

**Review by CagtayFabry — CHANGES_REQUESTED** [review]
good work refactoring the code! 👍´

I just added some minor comments to double check

File: testmodule/jobfile.py (hunk 1 above)
Comment (barisccolak) [reply testmodule/jobfile.py:22]: might be useful for #43

**Pull request merged**